packages/debian/entrypoint.sh

```diff
@@ -35,6 +35,22 @@ rpm2deb() {
     chown "${TARGET_UID}:${TARGET_GID}" * 
 }
 
+
+download_packages() {
+#<Package_name>=<Package_version>
+for pkg in $(cat list.txt)
+do
+        echo "$pkg" >> packages_list.txt
+        for dep in $(apt-rdepends "$pkg" | grep -E 'Depends|Recommends|Suggests' | cut -d " " -f4)
```

Comment:
I don't think we want the `Suggests`, and maybe not the `Recommends`.

Reply:
This is just temporary names waiting for a convention naming for these parameters

Reply:
I'm not talking about naming of variable here.
I'm talking about the deps we want to download.

Reply:
My bad, I misunderstood.
I corrected it !